superset-frontend/plugins/plugin-chart-echarts/src/Treemap/EchartsTreemap.tsx

```diff
@@ -84,6 +86,25 @@ export default function EchartsTreemap({
         handleChange([name]);
       }
     },
+    contextmenu: eventParams => {
```

Comment:
If you right-click on the dimension label for Treemap v2, the menu looks like it has a broken value:

<img width="353" alt="Screen Shot 2022-08-09 at 9 48 54 AM" src="https://user-images.githubusercontent.com/13007381/183699108-fa4b3539-05fe-4cbb-91ed-584241ba92d8.png">